packages/bot-web-ui/src/assets/sass/flyout.scss

```diff
@@ -118,9 +127,14 @@
             margin-bottom: 10px;
         }
         &-label {
-            font-weight: bold;
-            font-size: 18px;
-            line-height: 2em;
+            &--bold {
+                line-height: 2em;
+                font-weight: bold;
+                font-size: 18px;
+            }
+            &--small {
+                font-size: 14px;
+            }
```

Comment:
the label styles are only related to the Text component you can update it to Text and remove these styles
or if you need to have these class names please use rem instead of px

Reply:
I will use `em` for font-size.